pkg/remediator/watch/filteredwatcher.go

```diff
@@ -105,8 +105,8 @@ type filteredWatcher struct {
 // filteredWatcher implements the Runnable interface.
 var _ Runnable = &filteredWatcher{}
 
-// NewFiltered returns a new filtered watch initialized with the given options.
-func NewFiltered(_ context.Context, cfg watcherConfig) Runnable {
+// NewFiltered returns a new filtered watcher initialized with the given options.
+func NewFiltered(cfg watcherConfig) Runnable {
```

Comment:
Why did this take a context before in the first place? Why would a constructor take a context at all?